synapse/tests/test_lib_storm.py

```diff
@@ -297,28 +268,12 @@ class StormTest(s_t_utils.SynTest):
             self.len(1, nodes)
             self.eq(0x05060708, nodes[0].ndef[1])
 
-            # Sad paths where there are no nodes which match the specified values.
-            self.len(0, await core.nodes('test:guid | max :newp'))
-            self.len(0, await core.nodes('test:guid | min :newp'))
-
-            # Sad path for a form, not a property; and does not exist at all
-            with self.raises(s_exc.BadSyntax):
-                await core.nodes('test:guid | max test:newp')
-            with self.raises(s_exc.BadSyntax):
-                await core.nodes('test:guid | min test:newp')
-
-            # Ensure that max evaluates ival properties as the upper bound.
-            async with await core.snap() as snap:
-                node = await snap.addNode('test:guid', '*', {'tick': '2015',
```

Comment:
why is this commented out at all?

Reply:
err deleted

Reply:
they test min/max behavior over intervals

Reply:
( and the examples above are not bad syntax, they would raise NoSuchProp long before ever dispatching the command (and as such that functionality gets tested all over and doens't need to be tested again )

Reply:
move this into docs instead